src/test/java/org/wildfly/test/integration/vdx/standalone/InvalidCharactersTestCase.java

```diff
@@ -0,0 +1,156 @@
+                    + "        </subsystem>";
+
+    private static final String missingLastQuote =
+            "        <subsystem xmlns=\"urn:jboss:domain:webservices:2.0\">\n"
+                    + "            <wsdl-host>${jboss.bind.address:127.0.0.1}</wsdl-host>\n"
+                    + "            <modify-wsdl-address>true</modify-wsdl-address>\n"
+                    + "            <endpoint-config name=\"Standard-Endpoint-Config/>\n" //  <-- here is the missing end quote
+                    + "            <endpoint-config name=\"Recording-Endpoint-Config\">\n"
+                    + "                <pre-handler-chain name=\"recording-handlers\" protocol-bindings=\"##SOAP11_HTTP ##SOAP11_HTTP_MTOM ##SOAP12_HTTP ##SOAP12_HTTP_MTOM\">\n"
+                    + "                    <handler name=\"RecordingHandler\" class=\"org.jboss.ws.common.invocation.RecordingServerHandler\"/>\n"
+                    + "                </pre-handler-chain>\n"
+                    + "            </endpoint-config>\n"
+                    + "            <client-config name=\"Standard-Client-Config\"/>\n"
+                    + "        </subsystem>";
+
+    private String elementWithNationalCharacters = "<řebříček>obecný</řebříček>";
+
+    @BeforeClass
+    public static void prepareBrokenConfiguration() throws IOException {
+        try (Scanner scanner = new Scanner(standaloneXml);
+            PrintWriter missingStartOfElementStandaloneXmlPrintWriter = new PrintWriter(missingStartOfElementStandaloneXml.toFile());
+            PrintWriter spaceInXmlDeclarationStandaloneXmlPrintWriter = new PrintWriter(spaceInXmlDeclarationStandaloneXml.toFile());
+            PrintWriter missingLastQuoteStandaloneXmlPrintWriter = new PrintWriter(missingLastQuoteStandaloneXml.toFile())) {
+
+            boolean inWSSubsystem = false;
+            boolean wasInFirstLine = false;
+
+            while (scanner.hasNext()) {
+                String line = scanner.nextLine();
+
+                if (wasInFirstLine) {
+                    spaceInXmlDeclarationStandaloneXmlPrintWriter.println(line);
+                } else {
+                    wasInFirstLine = true;
+                    spaceInXmlDeclarationStandaloneXmlPrintWriter.println(" " + line);
+                }
+
+                if (line.contains("<subsystem xmlns=\"urn:jboss:domain:webservices")) {
+                    inWSSubsystem = true;
+
+                } else if (!inWSSubsystem) {
+                    missingStartOfElementStandaloneXmlPrintWriter.println(line);
+                    missingLastQuoteStandaloneXmlPrintWriter.println(line);
+
+                } else if (line.contains("</subsystem>")) {
+                    missingStartOfElementStandaloneXmlPrintWriter.println(missingStartOfElement);
+                    missingLastQuoteStandaloneXmlPrintWriter.println(missingLastQuote);
+                    inWSSubsystem = false;
+                }
+            }
+        }
+    }
```

Comment:
Static xml files could be placed in resources/configurations/standalone directory instead of dynamic generation prepareBrokenConfiguration() to $JBOSS_HOME/standalone/configuration/ directory. This seems to be quite heavy coding but it's minor thing.